src/js/view/controls/components/thumbnails.mixin.ts

```diff
@@ -0,0 +1,137 @@
+import { sortedIndex, property, bind } from 'utils/underscore';
```

Comment:
I wonder if there is a trick to getting git to see this as a move rather than a delete and add - is it just based on number of lines changed, added or removed?